### PR TITLE
Update reviewing.md

### PR DESCRIPTION
**Pull request by prince-chrismc**
Docs!

I've added two bullet points for the test package which I find I repeat fairly often.

Feedback always welcomed!

### CODE REVIEW

**Review by SSE4 — DISMISSED** [review]

File: docs/reviewing.md [hunk 1]
```diff
@@ -34,4 +34,19 @@ Prefer the following order of documented methods in python code (`conanfile.py`,
...
+## Test Package
+
+### Minimalistic Source Code
+
+The contents of `test_package.c` or `test_package.cpp` should be as minimal as possible, including a few headers at most with simple instatiation of objects to ensure linkage
+and dependencies are correct.
+
+### Verifying Components
+
+When components are defined in the `packge_info` in `conanfile.py` tthe following conditions are desired
```
Comment (uilianries) [inline docs/reviewing.md:49]:
```suggestion
When components are defined in the `packge_info` in `conanfile.py` the following conditions are desired
```

**Review by uilianries — COMMENTED** [review]
Great! Thanks @prince-chrismc !

**Review by uilianries — APPROVED** [review]
LGTM

File: docs/reviewing.md [hunk 2]
```diff
@@ -34,4 +34,19 @@ Prefer the following order of documented methods in python code (`conanfile.py`,
...
+## Test Package
+
+### Minimalistic Source Code
+
+The contents of `test_package.c` or `test_package.cpp` should be as minimal as possible, including a few headers at most with simple instatiation of objects to ensure linkage
+and dependencies are correct.
+
+### Verifying Components
+
+When components are defined in the `packge_info` in `conanfile.py` the following conditions are desired
```
Comment (intelligide) [inline docs/reviewing.md:49]:
```suggestion
When components are defined in the `package_info` in `conanfile.py` the following conditions are desired
```

**Review by intelligide — CHANGES_REQUESTED** [review]

**Comment by prince-chrismc** [comment]
I swear when I speak in English it sounds like the words which I can not spell 🗣️

**Review by intelligide — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Pull request merged**